WooCommerce/Classes/ViewRelated/Survey/SurveyViewController.swift

```diff
@@ -74,11 +74,10 @@ extension SurveyViewController {
                     .tagPlatform("ios")
                     .tagAppVersion(Bundle.main.bundleVersion())
 
-            case .productsM5Feedback:
-                return WooConstants.URLs.productsM4Feedback
+            case .productsVariationsFeedback:
+                return WooConstants.URLs.productsFeedback
                     .asURL()
                     .tagPlatform("ios")
-                    .tagProductMilestone("5")
```

Comment:
Maybe we need `product-milestone` = 6 for variations here?

cc @shiki who worked on previous version of this banner

Reply:
I don't exactly remember why we have this tag. Do you remember, @pmusolino? 

What I can deduce is that this would be useful to identify what product-related features are available when the merchant submitted the feedback. Let's say, the feedback is “I cannot add variations” and the tagged milestone is 5. With that, we'll know that the feedback is no longer valid because we have the new variations feature available now. 

But since app versions are now included in the tag (https://github.com/woocommerce/woocommerce-ios/pull/3796), I don't think this is necessary anymore. 

Also, we stopped using Milestone numbers at some point. 🤷

Reply:
Removed `tagProductMilestone` completely in 358d045, since it's unused now.

Reply:
I don't remember why we are using this tag 🤔 But it seems superfluous now.